Combobox.js

```diff
@@ -272,7 +305,8 @@ define([
 			}
 		},
 
-		refreshRendering: function (oldValues) {
+		/* jshint maxcomplexity: 17 */
+		refreshRendering: function (oldValues, justCreated) {
 			var updateReadOnly = false;
 			if ("list" in oldValues) {
 				this._initList();
```

Comment:
Don't you think we should use a flag to prevent multiple `_initList()` calls? This could cause the binding of multiple event listener. See `_prepareInput()` please.